appcelerator/titanium_mobile
# [TIMOB-26514] Android: Fixed 7.5.0.RC bug where theme applied to ActionBar/Toolbar title is lost when changing orientation

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-26514

**Summary:**
- Was hard-coded to use default "Widget_AppCompat_Toolbar" theme.
- Now fetches current theme's assigned "actionBarStyle" for `ActionBar`.
- Now fetches current theme's assigned "toolbarStyle" for `Toolbar`.

**Test:**
1. Run the test shown in [TIMOB-26514](https://jira.appcelerator.org/browse/TIMOB-26514).
2. Verify that top title bar text is white when rotating device.
